[0m22:00:06.530209 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002B124682E40>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002B12432F250>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002B1270F02D0>]}


============================== 22:00:06.539712 | 11e57578-874a-4da2-a735-8c87206b7ca1 ==============================
[0m22:00:06.539712 [info ] [MainThread]: Running with dbt=1.8.7
[0m22:00:06.541192 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\khilto01\\.dbt', 'fail_fast': 'False', 'version_check': 'True', 'log_path': 'logs', 'debug': 'False', 'warn_error': 'None', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'log_format': 'default', 'static_parser': 'True', 'invocation_command': 'dbt init', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'target_path': 'None', 'introspect': 'True', 'send_anonymous_usage_stats': 'True'}
[0m22:00:37.622764 [debug] [MainThread]: Starter project path: C:\Users\khilto01\.virtualenvs\cap-bikeshare-dbt\Lib\site-packages\dbt\include\starter_project
[0m22:00:37.665054 [info ] [MainThread]: 
Your new dbt project "cap_bike" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m22:00:37.665871 [info ] [MainThread]: Setting up your profile.
[0m22:02:14.387107 [info ] [MainThread]: Profile cap_bike written to C:\Users\khilto01\.dbt\profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m22:02:14.388602 [debug] [MainThread]: Command `dbt init` succeeded at 22:02:14.388402 after 128.13 seconds
[0m22:02:14.389150 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002B15069DF30>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002B12704CEF0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002B1505C9370>]}
[0m22:02:14.389711 [debug] [MainThread]: Flushing usage events
[0m11:02:15.300717 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000278673D2E40>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002786708F250>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000027869E202D0>]}


============================== 11:02:15.307507 | 62e35383-8127-4f92-953c-3777740d6983 ==============================
[0m11:02:15.307507 [info ] [MainThread]: Running with dbt=1.8.7
[0m11:02:15.308227 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\khilto01\\.dbt', 'version_check': 'True', 'debug': 'False', 'log_path': 'logs', 'fail_fast': 'False', 'warn_error': 'None', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'False', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt run', 'static_parser': 'True', 'introspect': 'True', 'target_path': 'None', 'log_format': 'default', 'send_anonymous_usage_stats': 'True'}
[0m11:02:15.308773 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path C:\Users\khilto01\Documents\git\cap-bikeshare\cap-bikeshare-dbt\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m11:02:15.309853 [debug] [MainThread]: Command `dbt run` failed at 11:02:15.309693 after 0.43 seconds
[0m11:02:15.310326 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000027869EBA060>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000027869D7F2F0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000027869DCABE0>]}
[0m11:02:15.310765 [debug] [MainThread]: Flushing usage events
[0m11:28:01.829177 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000014634106E40>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000014633DAF250>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000014636B702D0>]}


============================== 11:28:01.837291 | 8cd80316-7a15-4df1-b238-a845e5d9990c ==============================
[0m11:28:01.837291 [info ] [MainThread]: Running with dbt=1.8.7
[0m11:28:01.838869 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\khilto01\\.dbt', 'version_check': 'True', 'warn_error': 'None', 'log_path': 'logs', 'fail_fast': 'False', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'False', 'quiet': 'False', 'no_print': 'None', 'log_format': 'default', 'static_parser': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'target_path': 'None', 'invocation_command': 'dbt run', 'send_anonymous_usage_stats': 'True'}
[0m11:28:01.840459 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path C:\Users\khilto01\Documents\git\cap-bikeshare\cap-bikeshare-dbt\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m11:28:01.842369 [debug] [MainThread]: Command `dbt run` failed at 11:28:01.842082 after 0.27 seconds
[0m11:28:01.843147 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000014636C0A060>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000014636ACF2F0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000014636B1ABE0>]}
[0m11:28:01.843912 [debug] [MainThread]: Flushing usage events
